source/common/stats/stats_impl.h

```diff
@@ -412,8 +412,12 @@ class HistogramImpl : public Histogram, public MetricImpl {
 class HeapRawStatDataAllocator : public RawStatDataAllocator {
 public:
   // RawStatDataAllocator
+  typedef std::unordered_map<std::string, RawStatData*> StringRawDataMap;
```

Comment:
nit: since no other classes need to know about this typedef, do you think it makes sense to make it private?